app/code/RefreshMarkdownTask.php

```diff
@@ -94,11 +94,9 @@ private function cloneRepository(array $repository)
         $this->printLine("cloning " . $remote . "/" . $branch);
 
         chdir("{$path}/src");
-        exec("git clone -q git://github.com/{$remote}.git {$folder}_{$branch} --quiet");
+        exec("git clone -q https://github.com/{$remote}.git {$folder}_{$branch} --depth 1 --branch {$branch} --single-branch");
```

Comment:
Nice :) At one stage, I did look to see if you could only clone one directory (ie: docs) but that isn't possible.